nautobot/docs/development/getting-started.md

```diff
@@ -22,52 +36,135 @@ LICENSE.txt      development  mkdocs.yml  pyproject.toml           upgrade.sh
 NOTICE           dist         nautobot    scripts
 ```
 
-The Nautobot project utilizes three persistent git branches to track work:
+### About Remote Repos
 
-* `main` - Serves as a snapshot of the current stable release
-* `develop` - All development on the upcoming stable release occurs here
-* `feature` - Tracks work on an upcoming major release
+Git refers to remote repositories as *remotes*. When you make your initial clone of your fork, Git defaults to naming this remote `origin`. Throughout this documentation, the following remote names will be used:
 
-Typically, you'll base pull requests off of the `develop` branch, or off of `feature` if you're working on a new major release. **Never** merge pull requests into the `main` branch, which receives merged only from the `develop` branch.
+- `origin` - The default remote name used to refer to *your fork of Nautobot* 
+- `upstream` - The main remote used to refer to the *official Nautobot repository*
+
+### Setting up your Remotes
```

Comment:
It's not clear to me actually what we gain by adding this section. Are you planning to add more content about pulling from upstream as needed, etc.? As written currently I'm not seeing where we're using the `upstream` remote at all.

Reply:
Yeah. That's what I mentioned in the git commit. :P

Reply:
:reading-is-fundamental:

Reply:
I'll add a short section on branching to reference it.